plugin/hosts/setup_test.go

```diff
@@ -84,3 +84,77 @@ func TestHostsParse(t *testing.T) {
 		}
 	}
 }
+
+func TestHostsInlineParse(t *testing.T) {
+	tests := []struct {
+		inputFileRules      string
+		shouldErr           bool
+		expectedbyAddr      map[string][]string
+		expectedFallthrough bool
+	}{
+		{
+			`hosts inline example.org {
+                                10.0.0.1 example.org
+                                fallthrough
+                        }`,
+			false,
+			map[string][]string{
+				`10.0.0.1`: {
+					`example.org.`,
+				},
+			},
+			true,
+		},
+		{
+			`hosts inline example.org {
```

Comment:
`inline` signals a non-existent /etc/hosts file?

There is no IPv6 address in these tests, nor an alias: ip name alias